#goal definition
float32 altitude_setpoint
float32 duration
---
#result definition
bool finished
---
#feedback
float32 actual_alt